### ISSUE TITLE
[Feature] GPT-4o with audio support

### ISSUE THREAD

**Issue by codewithkenzo** [end-4/dots-hyprland#500]
Mr. end-4, I know you want it too

**Comment by H0mire** [comment]
Can't wait for OpenAI to release GPT-4o with stt and tts support. :o

**Comment by H0mire** [comment]
Yeah currently "Audio" is usually generated through a tts service, which you would have to integrate separately. OpenAI hinted that they will release the GPT 4o with Audio processing, which is basically native tts and stt without a separate model or service. This Results to a low latency like a normal human conversation and  capability to process emotional expression.
